pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -68,6 +70,100 @@ func (alloc *backfillAction) Execute(ssn *framework.Session) {
 			}
 		}
 	}
+
+	if ssn.ActionOptions == nil {
+		return
+	}
+
+	backfillEnabled, err := strconv.ParseBool(ssn.ActionOptions[conf.BackfillFlagName])
```

Comment:
refer to plugin's argument on how to get value.

Reply:
working on this one.